pandas/core/computation/expressions.py

```diff
@@ -35,6 +36,9 @@ def set_use_numexpr(v=True):
     global _USE_NUMEXPR
     if _NUMEXPR_INSTALLED:
         _USE_NUMEXPR = v
+    elif v:
+        warnings.warn("trying to set the option to use numexpr, "
+                      "but it is not installed!")
```

Comment:
I know I suggested this, but, I think this one will be triggered by default if you don't have numexpr installed? (just by importing pandas, I mean, not only when setting the option manually when you don't have it installed)